multi-arch-builders/provisioning/aarch64/main.tf

```diff
@@ -83,7 +127,7 @@ resource "aws_instance" "coreos-aarch64-builder" {
     Name = "${var.project}-${formatdate("YYYYMMDD", timestamp())}"
   }
   ami           = local.ami 
-  user_data     = file("coreos-aarch64-builder.ign")
+  user_data     = data.ct_config.butane.rendered
```

Comment:
Here we are using `rendered` instead of butane itself. Won't it lose some butane sugar?

Reply:
I think `.rendered` is just the rendered Ignition config (JSON). Any sugar from the butane config should have been now "rendered" into Ignition config.